GameOfLife/Calculations/Rules.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    class Rules
```

Comment:
Also, think about class naming. Should cell calculations go under class named Rules? Think about what the methods are doing. They are updating game field, creating next generation of cells. When i read the name Rules, that is not what comes to mind.

Reply:
Renamed to NextCellGeneration